select.java

```diff
@@ -0,0 +1,43 @@
+import java.sql.*;
+
+public class JavaMysqlSelectExample
+{
+
+  public static void main(String[] args)
+  {
+    try
+    {
+
+      String myDriver = "org.gjt.mm.mysql.Driver";
+      String myUrl = "jdbc:mysql://localhost/test";
+      Class.forName(myDriver);
+      Connection conn = DriverManager.getConnection(myUrl, "root", "");
+
+      String query = "SELECT * FROM *";
+
+      Statement st = conn.createStatement();
+      
+	  ResultSet rs = st.executeQuery(query);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code contains a resource that might not be closed properly. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *rs*. The resource is referenced in statements at the following lines: 18, 20, 23, 25, 26, 27, 28, 29, 30. The resource closure statement is at line: 35. There are other execution paths that do not contain closure statements, e.g., when exception is thrown by ResultSet.next. Either a) close *rs* in a try-finally block or b) close the resource by declaring *rs* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).